try-slang.js

```diff
@@ -307,49 +685,83 @@ var onRun = () => {
     const entryPointName = shaderType == SlangCompiler.RENDER_SHADER? "imageMain" : "printMain";
     const ret = compileShader(userSource, entryPointName, "WGSL");
 
-    // Recompile the pipeline.
-    if (ret.succ)
+    if (!ret.succ)
     {
-        const module = device.createShaderModule({code:ret.code});
-        computePipeline.createPipeline(module);
+        return;
     }
     else
     {
         toggleDisplayMode(HIDDEN_MODE);
         return;
     }
 
-    toggleDisplayMode(compiler.shaderType);
-    if (compiler.shaderType == SlangCompiler.PRINT_SHADER)
-    {
-        printResult();
-    }
-    else if (compiler.shaderType == SlangCompiler.RENDER_SHADER)
-    {
-        startRendering();
+    globalThis.hashedStrings = ret.hashedStrings;
```

Comment:
I don't quite understand here, it looks like the it won't go here anyway as it will return no matter what `ret.succ` is.

Reply:
Whoops! I think this happened when merging with upstream.. I put a return in one of the cases, and the merge created a return in the other case. 
Fixed now.